function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -749,8 +774,11 @@ public async Task<List<QueryValeuModel>> Query_entities(object filter)
         {
             var correlationId = Guid.NewGuid().ToString();
             var token = this.GetToken();
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? "purview-to-adb-demo-openlineage-purview"}.catalog.purview.azure.com/api";
-            var purviewSearchEndpoint = $"{baseurl}/search/query?api-version=2021-05-01-preview";
+            
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com